web-ng/src/app/components/drawing-kit/drawing-kit.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+  Copyright 2020 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<mat-button-toggle-group 
+class="drawing-buttons"
+[(value)] = "selectedValue">
+  <mat-button-toggle value = "{{pointValue}}" (click)="onButtonClick()">
```

Comment:
Please add a TODO here to replace with icons. Could you also please add a GitHub issue to track?